src/pages/home/report/ReportActionItemCreated.js

```diff
@@ -76,7 +77,13 @@ function ReportActionItemCreated(props) {
                         accessibilityLabel={props.translate('common.details')}
                         accessibilityRole="button"
                     >
-                        <RoomHeaderAvatars icons={icons} />
+                        <MultipleAvatars
+                            icons={icons}
+                            size={props.isLargeScreenWidth || (icons && icons.length < 3) ? CONST.AVATAR_SIZE.LARGE : CONST.AVATAR_SIZE.MEDIUM}
```

Comment:
I am wondering if these should be fixed inside the component? The size? @puneetlath do you think we'll need flexibility of setting this component wise?

Reply:
For workspace, it's always shown in the right-hand pane which is always the same-ish size right?

Reply:
That makes sense to me.